packages/core/src/card/card.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Card" />
+
+# Card
+
+## Installation
+
+```typescript
+import '@cds/core/card/register.js';
+```
+
+## Usage
+
+```html
+<cds-card>
+  <div cds-layout="vertical gap:md">
+    <div cds-text="section">
+      Card Title
+    </div>
+
+    <cds-divider cds-card-remove-margin></cds-divider>
+
+    <div cds-text="body" cds-layout="p-y:lg">
+      Content
+    </div>
+
+    <cds-divider cds-card-remove-margin></cds-divider>
+
+    <div cds-layout="horizontal gap:sm align:vertical-center">
+      <cds-button action="flat-inline">View</cds-button>
```

Comment:
Looking at the demo I think we need to remove the line height on the flat-inline button as well with the height. Its inheriting the parent line height which pushed the text down.

```css
:host([action='flat-inline']) {
  --height: auto;
  --padding: 0;
  line-height: 0;
```

![Screen Shot 2021-05-19 at 4 27 36 PM](https://user-images.githubusercontent.com/2021067/118887057-694f4880-b8bf-11eb-8ea6-aa01d379047d.png)

Reply:
https://github.com/vmware/clarity/pull/5970